### PR TITLE
Add Tizen Driver

### PR DESCRIPTION
**Pull request by flyofsky**
## Change list

Added Tizen Driver to support Appium in Tizen.
 
## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Details

This PR is to support Appium on the Tizen Platform.

### CODE REVIEW

File: appium-dotnet-driver/Appium/Tizen/TizenDriver.cs [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+        /// </summary>
+        /// <param name="service">the specified Appium local service</param>
+        /// <param name="desiredCapabilities">An <see cref="ICapabilities"/> object containing the desired capabilities.</param>
+        /// <param name="commandTimeout">The maximum amount of time to wait for each command.</param>
+        public TizenDriver(AppiumLocalService service, DesiredCapabilities desiredCapabilities, TimeSpan commandTimeout)
+            : base(service, SetPlatformToCapabilities(desiredCapabilities, Platform), commandTimeout)
+        {
+        }
+
+        public bool CompareImages(Image firstImage, Image secondImage, out Image resultImage)
```
Comment (mykola-mokhnach) [inline appium-dotnet-driver/Appium/Tizen/TizenDriver.cs]: Is this method really needed? Appium provides more advanced image comparison stuff based on OpenCV: https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/image-comparison.md

Comment (flyofsky) [reply appium-dotnet-driver/Appium/Tizen/TizenDriver.cs]: We need simple image compare function and don't want to send result image(first image).
I think this method support simple usage.

Comment (rookiejava) [reply appium-dotnet-driver/Appium/Tizen/TizenDriver.cs]: I agree with @mykola-mokhnach . Ok, we'll update it. Thanks.

Comment (flyofsky) [reply appium-dotnet-driver/Appium/Tizen/TizenDriver.cs]: Updated it. more review or merge plz

**Comment by flyofsky** [comment]
@SrinivasanTarget are you able to review and merge this?

**Comment by flyofsky** [comment]
@TikhomirovSergey are you able to review and merge this?

**Comment by jlipps** [comment]
@flyofsky we're getting reports that the solution doesn't compile with this merge. Are you able to double check and provide assistance?

**Comment by rookiejava** [comment]
@jlipps Yep. #240 is perfect.

**Pull request merged**